pkg/cmd/server/origin/master_config.go

```diff
@@ -434,6 +434,11 @@ func (c *MasterConfig) ImageImportControllerClient() *osclient.Client {
 	return c.PrivilegedLoopbackOpenShiftClient
 }
 
+// DeploymentConfigScaleClient returns the client used by the Scale subresource registry
+func (c *MasterConfig) DeploymentConfigScaleClient() *kclient.Client {
```

Comment:
> This is for use inside the API server (it's used by the registry code of DeploymentConfigs to fetch the corresponding ReplicationControllers). Does it still need to have a service account for that?

Not yet.  This would be ok for that.